src/App.jsx

```diff
@@ -1,9 +1,82 @@
 import './App.css';
+import React from "react";
+import {inventory, bestSellingTv} from "./constants/inventory.js";
+import {totalSold} from "./constants/soldHelper.js";
+import {totalBought} from "./constants/boughtHelper.js";
+import {leftToSell} from "./constants/leftToSellHelper.js";
+import {generateTelevisionTitle, addEuroSign} from "./constants/tvOptions.js";
+import {generateTelevisionSizeString} from "./constants/inchToCm.js";
+import {buttonHelper} from "./constants/buttonHelper.js";
+import checkLogo from "./assets/check.png"
+import minusLogo from "./assets/minus.png"
+import {bestForSport, cheapestFirst, mostSold} from "./constants/mostSold.js";
 
-function App() {
+
+export default function App() {
   return (
-    <h1>Begin hier met met maken van de applicatie!</h1>
-  )
-}
+      <div>
+          <h1>Tech it easy dashboard</h1>
+       <div className="productbox">
+        <div className="tile-green">
+          <p>Aantal verkochte producten</p>
+          <span>{totalSold(inventory)}</span>
+        </div>
+        <div className="tile-blue">
+          <p>Aantal ingekochte producten</p>
+          <span>{totalBought(inventory)}</span>
+        </div>
+        <div className="tile-red">
+          <p>Aantal te verkopen producten</p>
+          <span>{leftToSell(inventory)}</span>
+        </div>
+       </div>
+          <div><h3>Best verkochte tv</h3></div>
+          <div className="bestSellingTV">
+                      <div>
+                          <img src={bestSellingTv.sourceImg} alt={""} className="tvPicture" />
+                          {generateTelevisionTitle(bestSellingTv)}
+                      </div>
+                      <div>
+                          {addEuroSign(bestSellingTv.price)}
+                      </div>
+                      <div>
+                          {generateTelevisionSizeString(bestSellingTv)}
+                      </div>
+                      <div>
+                          {bestSellingTv.options.map((option, key)=>
+                              <React.Fragment key={key}>
+                                  <img src={option.applicable ? checkLogo : minusLogo} alt={""} className="logos" />
+                                 <span> {option.name} </span>
+                              </React.Fragment>
+                          )}
+                      </div>
+
+          </div>
+          <div>
+              <button onClick={ () => buttonHelper(mostSold())}>Meest verkocht eerst</button>
+              <button onClick={ () => buttonHelper(cheapestFirst())}>Goedkoopste eerst</button>
+              <button onClick={ () => buttonHelper(bestForSport())}>Meest geschikt voor sport eerst</button>
+          </div>
+          {inventory.map((television, key) => (
+              <React.Fragment key={key}>
 
-export default App
+                      <div className="listOfAllTv" >
+                          <img src={inventory.sourceImg} alt={"TV"} className="tvPicture" />
```

Comment:
Moet dit ook niet 
```suggestion
                          <img src={television.sourceImg} alt={"TV"} className="tvPicture" />
```
zijn?